.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -100,10 +100,15 @@ steps:
           - "heartbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "Heartbeat: Win-2022 Unit Tests"
+              context: "heartbeat: Win-2022 Unit Tests"
 
-      - label: ":ubuntu: Heartbeat Go Integration Tests"
-        command:  |
+      - label: ":ubuntu: Heartbeat Go (Module) Integration Tests"
+        command: |
+          set -euo pipefail
+          # defines the MODULE env var based on what's changed in a PR
+          source .buildkite/scripts/changesets.sh
+          defineModuleFromTheChangeSet heartbeat
```

Comment:
it's odd that we need to do that because there is no module directory under heartbeat: 

```
echo "~~~ Will run tests with env var MODULE=$MODULE"
cd heartbeat
mage goIntegTest
find: ‘heartbeat/module’: No such file or directory
```

https://buildkite.com/elastic/heartbeat/builds/4736#018f588a-18b8-419b-aa1c-e177306bef8c/140-149

Reply:
Additionally in the [BK output](https://buildkite.com/elastic/heartbeat/builds/4736#018f588a-18b8-419b-aa1c-e177306bef8c/150-151) I see:

```
Will run tests with env var MODULE=kubernetes
synthetics is available, but not invokable, command exited with bad code: sh: 1: elastic-synthetics: not found
```

which makes me wonder if we need to ensure this npm package is installed, as we do in https://github.com/elastic/beats/blob/main/.buildkite/x-pack/pipeline.xpack.heartbeat.yml#L60-L62 / https://github.com/elastic/beats/blob/main/.buildkite/x-pack/pipeline.xpack.heartbeat.yml#L27

Reply:
> Additionally in the [BK output](https://buildkite.com/elastic/heartbeat/builds/4736#018f588a-18b8-419b-aa1c-e177306bef8c/150-151) I see:
> 
> ```
> Will run tests with env var MODULE=kubernetes
> synthetics is available, but not invokable, command exited with bad code: sh: 1: elastic-synthetics: not found
> ```
> 
> which makes me wonder if we need to ensure this npm package is installed, as we do in https://github.com/elastic/beats/blob/main/.buildkite/x-pack/pipeline.xpack.heartbeat.yml#L60-L62 / https://github.com/elastic/beats/blob/main/.buildkite/x-pack/pipeline.xpack.heartbeat.yml#L27

Thanks for pointing this out, added package installation.